infra/aws/terraform/prow-build-cluster/resources/namespaces.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: k8s-infra-test-pods
```

Comment:
we don't really need this. Currently prow.k8s.io use only the test-pods namespace.